public/index.html

```diff
@@ -8,6 +8,13 @@
     <link rel="preload" href="%PUBLIC_URL%/images/background-v3@2x.jpg" as="image" />
     <link rel="preload" href="%PUBLIC_URL%/images/loading.gif" as="image" />
     <title>Decentraland</title>
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@decentraland" />
+    <meta name="twitter:creator" content="@decentraland" />
+    <meta property="og:url" content="%PUBLIC_URL%/" />
```

Comment:
I may be mistaken, but I think public_url will point to the permalink of the cdn, right?

Reply:
no, it shouldn't point to the cdn
I will change it